flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/functions/utils/UserDefinedFunctionUtils.scala

```diff
@@ -701,6 +701,18 @@ object UserDefinedFunctionUtils {
     getMethodSignatures(function, name).map(signatureToString).mkString(", ")
   }
 
+  /**
+    * Prints the first signature of methods with given name in a class
+    * and one signature consisting of DataType.
+    */
+  def signaturesToString(signature: Seq[LogicalType],
+                         function: UserDefinedFunction,
+                         name: String): String = {
+    val signatureArray = getMethodSignatures(function, name)(0).head +:
```

Comment:
The full signatures should be passed in as `signature` parameter. It's hack to extract the first argument of UDF. We already know the accumulator type in the context, we don't need to extract it from the UDF again.